_posts/2021-08-25-debezium-1-7-beta1-released.adoc

```diff
@@ -0,0 +1,27 @@
+---
+layout: post
+title:  Debezium 1.7.0.Beta1 Released
+date:   2021-08-25
+tags: [ releases, mysql, postgres, sqlserver, cassandra, oracle, db2, vitess, outbox, kafka ]
+author: rkerner
+---
+
+It's my pleasure to announce the first release of the Debezium 1.7 series, *1.7.0.Beta1*!
+
+This release brings https://docs.nats.io/developing-with-nats-streaming/streaming#nats-streaming-overview[NATS Streaming]
+support for Debezium Server along with many other fixes and enhancements. +
+Also this release is the first one tested with Apache Kafka 2.8.
+
++++<!-- more -->+++
+
+Also this release has a breaking change for the MySQL Connector. +
```

Comment:
Can we add something like this:

Also the Debezium container images for Apache Kafka and Kafka Connect (links tbd.) have been updated to version 2.8. This means that you can test Debezium with the new ZooKeeper-less mode for running Kafka (KIP-500 [link]). We'll share more details in a separate post shortly.